README.md

```diff
@@ -63,6 +63,12 @@ To see a list of all available tasks with their descriptions:
 $ crystal sam.cr help
 ```
 
+To skip a task or prerequisite task:
```

Comment:
Let's maybe make it a bit more clear:

To skip a task or prerequisite task specify it's path with "~" prefix